modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewDXPKey/pages/GenerateNewDXPKey.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable no-unused-vars */
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {ClaySelect} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import React, {useState} from 'react';
+import Button from '../../../../../common/components/Button';
+import {Radio} from '../../../../../common/components/Radio';
+import Layout from '../../../../../common/containers/setup-forms/Layout';
+
+const GenerateNewDXPKey = () => {
+	const [selected, setSelected] = useState();
+
+	const options = [
+		{
+			label: 'Option 1',
+			value: 1,
+		},
+		{
+			label: 'Option 2',
+			value: 2,
+		},
+		{
+			label: 'Option 3',
+			value: 3,
+		},
+	];
```

Comment:
This mocked values will be change until finish the story?

Reply:
Yeah! @jairmedeiros

Reply:
Yes! This values are here just because the API request will be in the next sub-task PR.